src/pretix/base/invoicing/transmission.py

```diff
@@ -119,6 +127,13 @@ def verbose_name(self):
         """
         raise NotImplementedError
 
+    @property
+    def testmode_supported(self) -> bool:
+        """
+        Whether testmode invoices may be passed to this provider.
+        """
+        return True
```

Comment:
```suggestion
        return False
```
I think providers should have to explicitly declare support, either by defaulting to False or raising an exception here.